### PR TITLE
Remove enable voting indication ff

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

To keep our codebase cleaner, we remove `ENABLE_VOTING_INDICATION` feature flag. The feature is already online, and this flag is no longer used. 

# Changes

- Remove the feature flag.
- Remove its usage.

# Tests

- Updated.

# Todos

- [ ] Add entry to changelog (if necessary).
Not necessary.

### CODE REVIEW

File: frontend/src/tests/lib/pages/SnsProposals.spec.ts
Comment (dskloetd) [inline frontend/src/tests/lib/pages/SnsProposals.spec.ts:1]: The indentation changes in many of the tests make it impossible to review the changes.
Would it be possible to keep the `describe` groups for now, to make reviewing possible.
And then unwrap the blocks in a separate PR where no other changes are made.

Comment (dskloetd) [reply frontend/src/tests/lib/pages/SnsProposals.spec.ts:1]: Thanks for pointing out the hide whitespace feature. I'll have another look tomorrow.

**Review by dskloetd — APPROVED** [review]
Rubber stamp

File: frontend/src/tests/lib/pages/SnsProposals.spec.ts
Comment (mstrasinskis) [reply frontend/src/tests/lib/pages/SnsProposals.spec.ts:1]: The `describe` blocks are back. It would be nice if you could take another quick look.

**Comment by dskloetd** [comment]
Please revert the last commit and then merge.
Because of the "hide whitespace" feature you pointed out, I was able to see the diffs without the need to restore the describe blocks and I approved based on that.

**Pull request merged**